tests/k8s_lib_injection/test_k8s_manual_inject.py

```diff
@@ -101,7 +101,61 @@ def test_inject_asm_admission_controller(self, test_k8s_instance):
 class TestAdmisionControllerProfiling:
     """Test profiling activation with the admission controller."""
 
-    def test_inject_asm_admission_controller(self, test_k8s_instance):
+    def _check_profiling_request_sent(self, agent_port):
+        """ Use test agent profiling endpoint to check 
+        if the profiling data has been sent by the injectect library. 
+        Checks the request made to the profiling endpoint (/profiling/v1/input)."""
+        response = requests.get(f"http://localhost:{agent_port}/test/session/requests")
```

Comment:
For host and container profiling, do we plan on keeping the full backend roundtrip, or do you want to eventually replace their logic too with this?

Reply:
In the near future we won't change this strategy for the host and container injection test